crates/codegen/runtime/generator/src/kinds/mod.rs

```diff
@@ -1,15 +1,52 @@
 use std::collections::BTreeSet;
+use std::rc::Rc;
 
+use codegen_ebnf::PlainWriter;
 use codegen_language_definition::model::{self, Identifier, Item, PredefinedLabel};
+use codegen_spec::SpecModel;
 use serde::Serialize;
 use strum::VariantNames;
 
+#[derive(Serialize, Ord, Eq, PartialEq, PartialOrd)]
+pub struct Kind {
+    id: Identifier,
+    documentation: String,
+}
+
+struct KindBuilder {
```

Comment:
suggestion on the rename, since this entire module is about kinds. This one here is about their EBNF:

```suggestion
struct EbnfBuilder {
```